linkerd/core/src/main/scala/io/buoyant/linkerd/Server.scala

```diff
@@ -119,12 +121,16 @@ class ServerConfig { config =>
       case Some(ps) => ApplicationProtocols.Supported(ps)
       case None => ApplicationProtocols.Unspecified
     }
-    Transport.ServerSsl(Some(SslServerConfiguration(
+    // The deprecated LegacyKeyServerEngineFactory allows us to accept PKCS#1 formatted keys.
+    // We should remove this and replace it with Netty4ServerEngineFactory once we no longer allow
+    // PKCS#1 keys.
+    @silent val sslServerEngine = SslServerEngineFactory.Param(LegacyKeyServerEngineFactory)
```

Comment:
I think we should probably just make a clean break in a future major release where we drop support for netty3 and require all keys to be pkcs#8